### PR TITLE
docstrings again

### PR DESCRIPTION
**Pull request by weakcamel**
* document models.del_none()
* change to Google convention for docstrings in setup.py
* add trove classifiers for the package in preparation for PyPI uploads

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -26,9 +29,12 @@ def read_list(fname):
...
     Load content of the file and split it into a list of lines.
 
-    :param fname: file name (or path relative to the project root)
-    :return: file content as a list of strings (one string per line)
-             with end of lines characters stripped off
+    Args:
+        fname (str): file name (or path relative to the project root)
+
+    Returns:
+        List[str]: file content (one string per line) with end of lines characters stripped off
```
Comment (dohnto) [inline setup.py]: Too long line, can we make it less than 80?

File: speechmatics/models.py [hunk 2]
```diff
@@ -8,6 +8,16 @@
 
 
 def del_none(dictionary):
+    """
+    Recursively delete all keys from the dictionary which are pointing to
```
Comment (dohnto) [inline speechmatics/models.py]: Don't like the word `pointing`. How about
```
Recursively delete all keys from the dictionary which values are None.
```

File: setup.py (hunk 1 above)
Comment (weakcamel) [reply setup.py]: Sure. I'll also add setup.py to the list of paths to lint (it currently isn't).

File: speechmatics/models.py (hunk 2 above)
Comment (weakcamel) [reply speechmatics/models.py]: Yes, that is better, thanks!

"Value of the key" is a bit ambiguous. Does "Recursively delete from the dictionary all _entries_ which values are None."  sound okay?

Comment (dohnto) [reply speechmatics/models.py]: sounds good

File: setup.py (hunk 1 above)
Comment (weakcamel) [reply setup.py]: done that + added `make lint` to CI config

**Comment by weakcamel** [comment]
Fixes pushed, @dohnto can you please verify?

**Comment by dohnto** [comment]
LGTM, merge/squash yourself please.

**Pull request merged**